certificate-service/pdfbox-generator/src/main/java/se/inera/intyg/certificateservice/pdfboxgenerator/pdf/CertificatePdfFillService.java

```diff
@@ -194,12 +194,18 @@ private void setFieldValue(PDAcroForm acroForm, PdfField field)
       final var extractedField = acroForm.getField(field.getId());
       validateField(field.getId(), extractedField);
 
-      if (extractedField instanceof PDTextField textField && textField.isMultiline()) {
-        final var textAppearance = new TextfieldAppearance(textField);
+      final var append = field.getAppend() != null && field.getAppend();
+
+      if (extractedField instanceof PDTextField textField && field.getAppearance() != null) {
+        textField.setDefaultAppearance(field.getAppearance());
```

Comment:
Not sure if or how this could be tested.

Reply:
Maybe you can get access to the field in the test and check that the appearance has been set/not set?